universe/base.go

```diff
@@ -127,14 +127,22 @@ func (a *Archive) RootNode(ctx context.Context,
 	})
 }
 
+type RootNodesQuery struct {
```

Comment:
These types should get added a commit earlier? IIUC they are used in commit 2 but added in commit 3.

Also, since both the RPC and the DB are using `int32` for these fields, the type here could be `int32` as well and eliminate some conversions elsewhere.

Reply:
> These types should get added a commit earlier? IIUC they are used in commit 2 but added in commit 3.

My bad, gonna rebase the commits